frontend/lib/account-settings/routes.tsx

```diff
@@ -0,0 +1,102 @@
+import { t } from "@lingui/macro";
+import React, { useContext } from "react";
+import { Link, Route } from "react-router-dom";
+import { AppContext } from "../app-context";
+import { TextualFormField } from "../forms/form-fields";
+import { SessionUpdatingFormSubmitter } from "../forms/session-updating-form-submitter";
+import { li18n } from "../i18n-lingui";
+import { NorentFullNameMutation } from "../queries/NorentFullNameMutation";
+import { bulmaClasses } from "../ui/bulma";
+import { EditableInfo } from "../ui/editable-info";
+import Page from "../ui/page";
+import { RequireLogin } from "../util/require-login";
+import { AccountSettingsRouteInfo } from "./route-info";
+
+type AccountSettingsContextType = {
+  routes: AccountSettingsRouteInfo;
+};
```

Comment:
I'm not sure if this is necessarily a good idea or not, but I really didn't want to have to tote around this as props everywhere.  (I expect the routes to change depending on the site that we attach it to.)